bin/nv

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const nv = require('../index')
+
+const args = require('yargs')
+  .command({
+    command: 'ls [versions...]',
+    desc: 'List Node.js versions',
+    builder: (yargs) => yargs.default('versions', 'lts_active'),
+    handler: (argv) => {
+      nv(argv.versions)
+        .then(result => {
+          result.forEach(r => {
+            console.log(JSON.stringify(r))
```

Comment:
```suggestion
            console.log(JSON.stringify(r, null, 2))
```

Reply:
@ljharb yeah I remembered that I ran `dependents` other day and it outputs like this:
```
$ node bin/dependents.js --pkg roi
Getting first 10 dependents of roi sorted by downloads
{"tenorjs":{"downloads":169,"forks":4,"stars":4,"watchers":2,"url":"github.com/Jinzulen/TenorJS"}}
{"clojars-client":{"downloads":9,"forks":0,"stars":1,"watchers":0,"url":"github.com/panther-js/clojars-client"}}
...
```
and it has an option to output as json (which uses the null replacement and the space parameters).

So I was just guessing we should have the same behavior here but :man_shrugging: 

Other thing is I was suggesting @wesleytodd to move the remaining change-requests as issues so that we can discuss and maybe enable other people to contrib etc. 

Anyway. whatever you folks have a final decision I update the PR :+1: